Add unit tests for wp_check_browser_version function

pbearne opened the pull request:
New unit tests have been implemented for the 'wp_check_browser_version' function in WordPress. This is done to verify the correctness of its output across different user agent strings, including when there's no agent and cases for Chrome on Windows and Android.

Trac ticket: 60828

pbearne committed:
Add unit tests for wp_check_browser_version function

New unit tests have been implemented for the 'wp_check_browser_version' function in WordPress. This is done to verify the correctness of its output across different user agent strings, including when there's no agent and cases for Chrome on Windows and Android.

mukeshpanchal27 commented on tests/phpunit/tests/admin/wp_check_browser_version.php:
```diff
@@ -0,0 +1,104 @@
+class Tests_dashboard_wp_check_browser_version extends WP_UnitTestCase {
+
+	public function set_up() {
+		/** Load WordPress dashboard API */
```
```suggestion
		/* Load WordPress dashboard API */
```

pbearne committed:
Add unit tests for wp_check_browser_version function

New unit tests have been implemented for the 'wp_check_browser_version' function in WordPress. This is done to verify the correctness of its output across different user agent strings, including when there's no agent and cases for Chrome on Windows and Android.

pbearne committed:
Update tests/phpunit/tests/admin/wp_check_browser_version.php

Co-authored-by: Mukesh Panchal <mukeshpanchal27@users.noreply.github.com>

Commit:
Rename test file and align array properties

The test file wp_check_browser_version.php has been renamed to wpCheckBrowserVersion.php to follow proper naming standards. Additionally, the properties within an array in the test file have been re-aligned for improved code readability.
